packages/client/src/pages/RecommendationsPage/RecommendationsTable/RecommendationsTable.tsx

```diff
@@ -108,6 +155,11 @@ const RecommendationsTable: FunctionComponent<RecommendationsTableProps> = ({
             row: classes.row,
           }}
           onRowClick={handleRowClick}
+          pageSize={pageSize}
+          components={{
+            Toolbar: CustomPagination(handleChange),
+            Pagination: CustomPagination(handleChange),
+          }}
```

Comment:
After making a change to the component declaration as described in the previous comment, the handleChange function can be assigned to the component separately like this: 
```suggestion
        components={{
            Toolbar: CustomPagination,
            Pagination: CustomPagination,
          }}
          componentsProps={{
             Toolbar: { handleChange }, // or { handleChange: handleChange }
             Pagination: { handleChange }, // or { handleChange: handleChange }
  }}
```